examples/mpi_call_optimization/Standard-VLE/CO2_Merker_220K/liq/run01/cp_binary-10.restart.header.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <mardyn version="20100525" >
 	<headerinfo>
-		<time>1.823670000000202e+01</time>
+		<time>1.823670000017852e+03</time>
 		<length>
-			<x>5.457241500000000e+01</x> <y>5.457241500000000e+01</y> <z>5.457241500000000e+01</z>
+			<x>5.000000000000000e+01</x> <y>5.000000000000000e+01</y> <z>5.000000000000000e+01</z>
 		</length>
-		<number>2048</number>
+		<number>1994</number>
```

Comment:
why did you change this?

Reply:
This is a restart file, which is the output of one simulation and then is used for the next bigger simulation.